code/modules/materials/_materials.dm

```diff
@@ -333,11 +334,15 @@ INITIALIZE_IMMEDIATE(/obj/effect/gas_overlay)
 /decl/material/proc/products_need_process()
 	return (radioactivity>0) //todo
 
+
+//Clausius–Clapeyron relation
+/decl/material/proc/get_boiling_temp(var/pressure = ONE_ATMOSPHERE)
+	return (1 / (1/boiling_point) - ((R_IDEAL_GAS_EQUATION * log(pressure / ONE_ATMOSPHERE)) / (latent_heat * molar_mass)))
```

Comment:
`boiling_point` might be 0 or null - should probably return INFINITY if it's not >0.